core/src/test/resources/jenkins/monitor/os-release-centos-7

```diff
@@ -0,0 +1,16 @@
+NAME="CentOS Linux"
+VERSION="7 (Core)"
+ID="centos"
+ID_LIKE="rhel fedora"
+VERSION_ID="7"
+PRETTY_NAME="CentOS Linux 7 (Core)"
+ANSI_COLOR="0;31"
+CPE_NAME="cpe:/o:centos:centos:7"
+HOME_URL="https://www.centos.org/"
+BUG_REPORT_URL="https://bugs.centos.org/"
+
+CENTOS_MANTISBT_PROJECT="CentOS-7"
+CENTOS_MANTISBT_PROJECT_VERSION="7"
+REDHAT_SUPPORT_PRODUCT="centos"
+REDHAT_SUPPORT_PRODUCT_VERSION="7"
+
```

Comment:
```suggestion
```

Reply:
I would rather not make this change.

The file is copied from the [CentOS 7 container image](https://hub.docker.com/r/jenkins/jenkins/tags?page=1&name=centos7) that we use for the Jenkins controller image.  Removing the empty line will make the test data file different than the file on the actual operating system.  It would be harmless in this case, since the test does not depend on that blank line, but I think it is better to keep the test data aligned with the file as it exists on CentOS 7.